_api-reference/list/list-indices.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: default
+title: List Indices 
+parent: List API
+nav_order: 25
+has_children: false
+---
+
+# List Indices
+**Introduced 2.18**
+{: .label .label-purple }
+
+The List Indices operation provides the following information related to indexes in a paginated format: 
+
+- The amount of disk space the index is using. 
+- The number of shards the index contains. 
+- The index's health status.
+
+## Path and HTTP methods
+
+```json
+GET _list/indices
+GET _list/indices/<index>
+```
+
+## Query parameters
+
+Parameter | Type | Description
+:--- | :--- | :---
+`bytes` | Byte size | Specifies the units for the byte size, for example, `7kb` or `6gb`. For more information, see [Supported units]({{site.url}}{{site.baseurl}}/opensearch/units/).
+`health` | String | Limits indexes based on their health status. Supported values are `green`, `yellow`, and `red`.
+`include_unloaded_segments` | Boolean | Whether to include information from segments not loaded into memory. Default is `false`.
+`cluster_manager_timeout` | Time | The amount of time to wait for a connection to the cluster manager node. Default is 30 seconds.
+`pri` | Boolean | Whether to return information only from the primary shards. Default is `false`.
+`time` | Time | Specifies the units for time, for example, `5d` or `7h`. For more information, see [Supported units]({{site.url}}{{site.baseurl}}/opensearch/units/).
+`expand_wildcards` | Enum | Expands wildcard expressions to concrete indexes. Combine multiple values with commas. Supported values are `all`, `open`, `closed`, `hidden`, and `none`. Default is `open`.
+`next_token` | String | Fetches the next page of indexes. When `null` provides the first page of indexes. Default is `null`. 
```

Comment:
"Fetches the next page of indexes when `null` provides the first page of indexes"?